valhalla/baldr/graphreader.h

```diff
@@ -452,6 +452,13 @@ class GraphReader {
    */
   virtual bool DoesTileExist(const GraphId& graphid) const;
 
+  /**
+   * Test if traffic tiles exist.   *
+   */
+  bool DoTrafficTilesExist() {
+    return !tile_extract_->traffic_tiles.empty();
+  }
```

Comment:
open for other ways..

Reply:
maybe a less perscriptive name like `HasLiveTraffic`